app/helpers/opengraph_helper.rb

```diff
@@ -0,0 +1,25 @@
+module OpengraphHelper
+  include Comfy::CmsHelper
+
+  # Methods which include a fallback if the requested value is blank
+  def og_description
+    social_desc = cms_fragment_content(:social_description, @cms_page)
+    summary = cms_fragment_content(:summary, @cms_page)
+    fallback_summary = summary.blank? ? t('meta.site.description') : summary
+    social_desc.blank? ? fallback_summary : social_desc
+  end
+
+  def og_image
+    image = cms_fragment_content(:image, @cms_page).attachments.first
+    path_to_image = URI.join(root_url, helpers.url_for(image))
+    fallback_image = URI.join(root_url, helpers.image_path(t('meta.image')))
+    image.blank? ? fallback_image : path_to_image
+  end
+
+  def og_title
+    social_title = cms_fragment_content(:social_title, @cms_page)
+    title = @cms_page.label
+    fallback_title = title.blank? ? t('meta.site.title') : title
+    social_title.blank? ? fallback_title : social_title
+  end
+end
```

Comment:
@stanleypliu I wonder if this is almost the same thing that the ComfyOpengraph class in `lib/modules` was trying to achieve?
If not, perhaps we could still amend or make use of that already existing class instead of creating a separate helper?

Reply:
More specifically, I would also suggest to check for blanks early within the methods as otherwise the code will throw an exception for all non CMS pages when trying to fetch the label for example, as @cms_page will be nil